catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvMetacardTransformer.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.csv;
+
+import static ddf.catalog.transformer.csv.common.CsvTransformer.createResponse;
+import static ddf.catalog.transformer.csv.common.CsvTransformer.writeMetacardsToCsv;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.BinaryContent;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transform.MetacardTransformer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class CsvMetacardTransformer implements MetacardTransformer {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(CsvMetacardTransformer.class);
+
+  @Override
+  public BinaryContent transform(Metacard metacard, Map<String, Serializable> arguments)
```

Comment:
❓ Alternatively we could call the query response transformer by constructing a `SourceResponse`. That way we don't have to duplicate any logic.

Reply:
My opinion is that I prefer calling the commons code